src/lib/canonical-title-map.js

```diff
@@ -14,6 +14,17 @@ export default function(titleMap: Array<any>, originalEnum?: any) {
       });
     }
     return canonical;
+  } else if (originalEnum) {
+    const canonical = [];
```

Comment:
This definition of canonical doesn't seem to be used. I think it is safe to remove it.